api/src/main/java/com/datatorrent/api/StatsListener.java

```diff
@@ -113,6 +113,8 @@
     long getLatencyMA();
 
     List<OperatorResponse> getOperatorResponse();
+
+    String getOperatorName();
```

Comment:
Why operator name is part of BatchedOperatorStats when operator name does not change?